packages/framework/ai-collab/src/explicit-strategy/agentEditReducer.ts

```diff
@@ -63,7 +78,10 @@ function populateDefaults(
 	}
 }
 
-function getSchemaIdentifier(content: TreeEditValue): string | undefined {
+/**
+ * Gets the schema identifier of the given content, including primitive values.
+ */
+export function getSchemaIdentifier(content: TreeEditValue): string | undefined {
```

Comment:
Not for this PR, but the below code should also probably explicitly check for Map nodes (and throw a usage error).

Reply:
oh? `isFluidHandle()` does not cover the Map node?

Reply:
`isFluidHandle` is specifically about the `IFluidHandle` type, which is used to embed references to other DDSs under SharedTree nodes. It won't handle any other cases (like ObjectNode, ArrayNode, MapNode, etc.).

Reply:
I see. adding 
~~~
if (isNode(content)) {
				throw new UsageError("Nodes are not currently supported in this context");
			}
~~~
here